src/Core/Framework/Script/Execution/ScriptExecutor.php

```diff
@@ -43,11 +42,6 @@ public function __construct(
 
     public function execute(Hook $hook): void
     {
-        // @deprecated tag:v6.7.0 - remove if condition
-        if (EnvironmentHelper::getVariable('DISABLE_EXTENSIONS', false)) {
```

Comment:
this looks weird, as it is not a feature flag as the others, but a runtime flag to ignore all extensions, not sure if or why it should now be removed
@shyim do you know?